test/unit/webgl/p5.Geometry.js

```diff
@@ -144,4 +144,110 @@ suite('p5.Geometry', function() {
       assert.equal(geom._addJoin.callCount, 0);
     });
   });
+
```

Comment:
Should we test that a variety of materials also still work correctly with the builder?

Reply:
I think if the resulting geometry is limited to only being able to use a single material (which probably makes sense), we should note that in the documentation. I'm fine with that limitation and don't think there'd be an easy or scalable way around that anyway.

But even in the case of a single material on the geometry, lets just double check that all the prefab materials (specular, ambient, etc...) are working correctly.

Reply:
Good call, I added a test and noticed an issue! Previously, all fills were getting stored in `vertexColors`, which override the `fill()` set before drawing  the geometry with `model()`. We don't have a good way of optionally saying "ignore the vertex colors for this draw" so for now I've added some code to not store the vertex colors if every vertex is the same color.